core/templates/pages/exploration-editor-page/history-tab/services/version-tree.service.ts

```diff
@@ -81,14 +83,15 @@ export class VersionTreeService {
    * The parent of the root (version 1) is -1.
    */
   getVersionTree(): {} {
-    if (this._treeParents === null) {
+    if (this._treeParents === undefined) {
```

Comment:
Why us this still needed?

Reply:
`_treeParents` is still uninitialized (undefined) at the top, so we need to add a check